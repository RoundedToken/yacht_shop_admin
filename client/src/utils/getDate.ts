export function getDate(isoDate: string): string {
    const date = new Date(isoDate);

    return date.toLocaleString('default', {
        day: '2-digit',
        month: 'short',
        year: 'numeric',
        hour: 'numeric',
        minute: 'numeric',
        hour12: false,
    });
}
