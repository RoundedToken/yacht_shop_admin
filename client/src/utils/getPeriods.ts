const toIso = (mils: number) => {
    const now = Date.now();

    return new Date(now - mils).toISOString();
};
const getThisWeek = () => {
    const today = new Date();

    const dayOfWeek = today.getDay() || 7;

    const monday = new Date(today.getTime() - (dayOfWeek - 1) * 24 * 60 * 60 * 1000);

    return monday.toISOString();
};

const getThisMonth = () => {
    const today = new Date();

    const year = today.getFullYear();
    const month = today.getMonth();

    const firstDay = new Date(year, month, 1, 23, 59);

    return firstDay.toISOString();
};

const getThisYear = () => {
    const today = new Date();

    const year = today.getFullYear();

    const firstDay = new Date(year, 0, 1, 23, 59);

    return firstDay.toISOString();
};

export const getPeriods = () => {
    const today = new Date().toISOString();
    const thisWeek = getThisWeek();
    const week = toIso(1000 * 60 * 60 * 24 * 7);
    const month = toIso(1000 * 60 * 60 * 24 * 30);
    const thisMonth = getThisMonth();
    const thisYear = getThisYear();
    const year = toIso(1000 * 60 * 60 * 24 * 365);

    return { week, month, year, today, thisWeek, thisMonth, thisYear };
};
