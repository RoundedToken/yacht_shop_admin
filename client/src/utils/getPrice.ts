export const getPrice = (price: number) => {
    const formatter = new Intl.NumberFormat('et-EE', {
        style: 'currency',
        currency: 'EUR',
    });
    return formatter.format(price);
};
