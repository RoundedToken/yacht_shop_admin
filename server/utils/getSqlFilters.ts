import { IFilters } from '../types';

export const getSqlFilters = (filters: string, isProducts: boolean = false) => {
    const filtersObj = JSON.parse(filters) as IFilters;
    const productId = filtersObj.product;

    const filterStr =
        Object.entries(filtersObj)
            .filter((filter) => filter[1] !== null && filter[1] !== '')
            .map((filter) => {
                const key = filter[0];
                const value = filter[1];

                if (key === 'dateFrom') {
                    return `ord.DateEnter >= CAST('${value}T00:00:00' AS datetime)`;
                } else if (key === 'dateTo') {
                    return `ord.DateEnter <= CAST('${value}T23:59:59' AS datetime)`;
                } else if (key === 'status') {
                    return `ord.status IN (${value.join(',')})`;
                } else if (key === 'isPaid') {
                    return `ord.paid = ${value ? 1 : 0}`;
                } else if (key === 'product') {
                    if (isProducts) return '1=1';
                    else return `ordStr.tovar = ${value}`;
                } else if (key === 'client') {
                    return `ord.clientid = ${value}`;
                } else {
                    return `STUFF((SELECT DISTINCT ', ' + oplata.type  
                    FROM Oplata 
                    WHERE Oplata.orderid = ord.orderid  
                    FOR XML PATH('')), 1, 2, '') LIKE '%${value}%'`;
                }
            })
            .join(' AND ') || '1=1';

    return { filterStr, productId };
};
