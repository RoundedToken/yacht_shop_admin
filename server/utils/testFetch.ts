export class testFetch {
    private endpoint: string;

    constructor(endpoint: string) {
        this.endpoint = endpoint;
    }

    async getData(params: Record<string, string>) {
        const baseUrl = 'http://192.168.1.37:5000/api';
        const url = baseUrl + this.endpoint + '?' + new URLSearchParams(params);

        const response = await fetch(url);

        return await response.json();
    }
}
