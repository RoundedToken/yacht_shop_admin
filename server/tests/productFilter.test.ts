import { describe, expect, test } from 'bun:test';
import { testFetch } from '../utils/testFetch';

describe('Product Filter', () => {
    const getSales = new testFetch('/get_sales');

    test('Get Sales product: 58160', async () => {
        const params = {
            pageIndex: '0',
            pageSize: '10',
            sortId: 'Id',
            sortType: 'DESC',
            filters:
                '{"dateFrom":"2022-06-01","dateTo":"2022-08-04","payType":null,"status":[1,-1],"isPaid":false,"product":58160,"client":null}',
        };
        const data = await getSales.getData(params);

        expect(data).toMatchSnapshot();
    });

    test('Get Products product: 58169', async () => {
        const params = {
            pageIndex: '0',
            pageSize: '10',
            sortId: 'Id',
            sortType: 'DESC',
            filters:
                '{"dateFrom":"2022-06-01","dateTo":"2022-08-04","payType":null,"status":[1,-1],"isPaid":false,"product":58169,"client":null}',
        };
        const data = await getSales.getData(params);

        expect(data).toMatchSnapshot();
    });
});
