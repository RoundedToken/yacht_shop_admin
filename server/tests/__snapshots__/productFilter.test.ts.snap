// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Filter Get Sales product: 58160 1`] = `
{
  "clients": [
    {
      "id": null,
      "name": null,
    },
    {
      "id": 245,
      "name": "Щербаков Антон Васильевич",
    },
    {
      "id": 627,
      "name": "Sailtech Oy (Финляндия)",
    },
    {
      "id": 676,
      "name": "Maritim-Finland",
    },
    {
      "id": 940,
      "name": "web",
    },
    {
      "id": 1108,
      "name": "TREM",
    },
    {
      "id": 1702,
      "name": "MarineTech (Germany)",
    },
    {
      "id": 2766,
      "name": "Optiparts",
    },
    {
      "id": 2773,
      "name": "Windex (Sweden)",
    },
    {
      "id": 9901,
      "name": "Anton Scherbakov",
    },
    {
      "id": 11182,
      "name": "_jamuuk",
    },
    {
      "id": 11183,
      "name": "PJK",
    },
    {
      "id": 11185,
      "name": "Regate (LV)",
    },
    {
      "id": 11186,
      "name": "TommyyLyy",
    },
    {
      "id": 11187,
      "name": "Pro Marine Trade",
    },
    {
      "id": 11188,
      "name": "Татьяна Каверина (Armastus)",
    },
    {
      "id": 11189,
      "name": "NORA LII SOPRADE SELTS",
    },
    {
      "id": 11190,
      "name": "Andres",
    },
    {
      "id": 11191,
      "name": "Nautix",
    },
    {
      "id": 11192,
      "name": "Torivallavalitsus",
    },
    {
      "id": 11193,
      "name": "Ninne",
    },
    {
      "id": 11194,
      "name": "Urmas Varris",
    },
    {
      "id": 11195,
      "name": "WindWalker",
    },
    {
      "id": 11196,
      "name": "Via Grupp OU (Командор)",
    },
    {
      "id": 11201,
      "name": "Tõnis",
    },
    {
      "id": 11202,
      "name": "Jaanus",
    },
    {
      "id": 11203,
      "name": "Meister Arle OU",
    },
    {
      "id": 11204,
      "name": "Prime Marine Service OU",
    },
    {
      "id": 11205,
      "name": "Viking",
    },
    {
      "id": 11206,
      "name": "Saku",
    },
    {
      "id": 11207,
      "name": "SineneLind",
    },
    {
      "id": 11208,
      "name": "Alex Tido",
    },
    {
      "id": 11209,
      "name": "AZZURRA",
    },
    {
      "id": 11210,
      "name": "квас на пиво",
    },
    {
      "id": 11211,
      "name": "MIA (Tallin)",
    },
    {
      "id": 11212,
      "name": "MINNI",
    },
    {
      "id": 11213,
      "name": "Jonпgert OU",
    },
    {
      "id": 11214,
      "name": "MIA Pärnu",
    },
    {
      "id": 11217,
      "name": "Tauno Teluk",
    },
    {
      "id": 11219,
      "name": "Капитан",
    },
    {
      "id": 11220,
      "name": "Johannes  Atlandi Võlg",
    },
    {
      "id": 11221,
      "name": "Nimi",
    },
    {
      "id": 11222,
      "name": "Viahooldus ОÜ",
    },
    {
      "id": 11223,
      "name": "Kaupmees",
    },
    {
      "id": 11224,
      "name": "Reino",
    },
    {
      "id": 11225,
      "name": "Rubit OU",
    },
    {
      "id": 11226,
      "name": "Aleksey  Rusetsky",
    },
    {
      "id": 11227,
      "name": "Corinthian Events OÜ",
    },
    {
      "id": 11228,
      "name": "Ronstan DK",
    },
    {
      "id": 11229,
      "name": "Narva-Joesuu PurjeSpoordiKool",
    },
    {
      "id": 11230,
      "name": "Saaremaa Meri Spordiselts",
    },
    {
      "id": 11231,
      "name": "Tallux Orto",
    },
    {
      "id": 11232,
      "name": "Uku Kuusk",
    },
    {
      "id": 11234,
      "name": "Bainbridge Inc (UK)",
    },
    {
      "id": 11235,
      "name": "BetaMarine (UK)",
    },
    {
      "id": 11236,
      "name": "Change Consulting OÜ",
    },
    {
      "id": 11237,
      "name": "Rait Maru",
    },
    {
      "id": 11239,
      "name": "Žaklar OU",
    },
    {
      "id": 11240,
      "name": "Comar Systems Limited",
    },
    {
      "id": 11241,
      "name": "Kaidor",
    },
    {
      "id": 11242,
      "name": "MADEN",
    },
    {
      "id": 11243,
      "name": "Ropebox Eesti OÜ",
    },
    {
      "id": 11244,
      "name": "Muri",
    },
    {
      "id": 11245,
      "name": "Carrestoration OU",
    },
    {
      "id": 11246,
      "name": "Siren Sailing OU",
    },
    {
      "id": 11247,
      "name": "Episood OU",
    },
    {
      "id": 11248,
      "name": "Akster Grupp OU",
    },
    {
      "id": 11249,
      "name": "Alter",
    },
    {
      "id": 11250,
      "name": "Ronstan AUS",
    },
    {
      "id": 11251,
      "name": "Innon Finance OU",
    },
    {
      "id": 11252,
      "name": "Lettore OÜ",
    },
    {
      "id": 11253,
      "name": "Navideep Services OÜ",
    },
    {
      "id": 11254,
      "name": "Athaman (Polska)",
    },
    {
      "id": 11255,
      "name": "Lõbusõit OÜ",
    },
    {
      "id": 11256,
      "name": "Aleksei Mikhalev",
    },
    {
      "id": 11257,
      "name": "Ropebox OÜ",
    },
    {
      "id": 11258,
      "name": "Kaubakeskus OÜ",
    },
    {
      "id": 11259,
      "name": "Rein Ottoson OÜ",
    },
    {
      "id": 11260,
      "name": "Robert",
    },
    {
      "id": 11261,
      "name": "Merema OÜ",
    },
    {
      "id": 11262,
      "name": "Alexander Rusetsky",
    },
    {
      "id": 11263,
      "name": "Natalja Rusetskaya",
    },
    {
      "id": 11264,
      "name": "Jahtklubi Dago",
    },
    {
      "id": 11265,
      "name": "Elev Luik",
    },
    {
      "id": 11266,
      "name": "_ЗХ2023",
    },
    {
      "id": 11267,
      "name": "Vsevolod pyramidaprint",
    },
    {
      "id": 11268,
      "name": "Emerald",
    },
    {
      "id": 11270,
      "name": "Strand Sailing Management",
    },
    {
      "id": 11271,
      "name": "Tuule",
    },
    {
      "id": 11272,
      "name": "Meelis Kojamess (Баба Яга)",
    },
    {
      "id": 11273,
      "name": "Andrea FE64",
    },
    {
      "id": 11274,
      "name": "Reilut OÜ",
    },
    {
      "id": 11275,
      "name": "Pärnu cruises",
    },
    {
      "id": 11280,
      "name": "Madis Parts",
    },
    {
      "id": 11281,
      "name": "Tera ehitus OU",
    },
    {
      "id": 11282,
      "name": "Korkvender OU",
    },
    {
      "id": 11291,
      "name": "STRAND PROJEKT OÜ",
    },
    {
      "id": 11292,
      "name": "RTR Tehnika",
    },
    {
      "id": 11293,
      "name": "Eprodukt  OÜ",
    },
    {
      "id": 11294,
      "name": "Madis Parts",
    },
    {
      "id": 11295,
      "name": "SeaKayaking",
    },
    {
      "id": 11296,
      "name": "8 Motor Oy",
    },
    {
      "id": 11297,
      "name": "SKELTMER OÜ",
    },
    {
      "id": 11298,
      "name": "Jäälind",
    },
    {
      "id": 11299,
      "name": "MattChem (France)",
    },
    {
      "id": 11300,
      "name": "Car Restoration OU",
    },
    {
      "id": 11301,
      "name": "Valgeranna Seikluspark OÜ",
    },
    {
      "id": 11302,
      "name": "Kaif OÜ",
    },
    {
      "id": 11303,
      "name": "Julechka",
    },
    {
      "id": 11304,
      "name": "Longboat",
    },
    {
      "id": 11305,
      "name": "SAILING MASSIVE  OÜ",
    },
    {
      "id": 11306,
      "name": "AB Technology",
    },
    {
      "id": 11307,
      "name": "Norman Saarso",
    },
    {
      "id": 11308,
      "name": "Tuisu OÜ",
    },
    {
      "id": 11309,
      "name": "Fairaway",
    },
    {
      "id": 11311,
      "name": "Alibi OÜ",
    },
    {
      "id": 11312,
      "name": "Dirham motorpaad",
    },
    {
      "id": 11313,
      "name": "Niined group",
    },
    {
      "id": 11314,
      "name": "Magna Fide",
    },
    {
      "id": 11315,
      "name": "Корячкин",
    },
    {
      "id": 11317,
      "name": "Konstrado",
    },
    {
      "id": 11318,
      "name": "Vektom OÜ",
    },
    {
      "id": 11319,
      "name": "Kalev Jahtklubi",
    },
    {
      "id": 11320,
      "name": "Marta Vintage OÜ",
    },
  ],
  "orders": [
    {
      "allSum": -2830.5905,
      "clientName": "Ronstan AUS",
      "date": "2022-07-01T19:13:00.000Z",
      "id": 205260,
      "isPaid": false,
      "paidSum": null,
      "payType": null,
      "positionCount": 22,
      "productsCount": 8,
      "productsSum": -198.9096,
      "status": "покупка",
    },
    {
      "allSum": -708.9206,
      "clientName": "Ropebox Eesti OÜ",
      "date": "2022-07-01T12:27:00.000Z",
      "id": 205155,
      "isPaid": false,
      "paidSum": -708.92,
      "payType": "Б/н",
      "positionCount": 9,
      "productsCount": 2,
      "productsSum": -62.1592,
      "status": "покупка",
    },
  ],
  "pagesCount": 1,
  "rowsCount": 2,
  "totalAllSum": -3539.5111,
  "totalPaidSum": -708.92,
  "totalProductSum": -261.0688,
}
`;

exports[`Product Filter Get Products product: 58169 1`] = `
{
  "clients": [
    {
      "id": null,
      "name": null,
    },
    {
      "id": 245,
      "name": "Щербаков Антон Васильевич",
    },
    {
      "id": 627,
      "name": "Sailtech Oy (Финляндия)",
    },
    {
      "id": 676,
      "name": "Maritim-Finland",
    },
    {
      "id": 940,
      "name": "web",
    },
    {
      "id": 1108,
      "name": "TREM",
    },
    {
      "id": 1702,
      "name": "MarineTech (Germany)",
    },
    {
      "id": 2766,
      "name": "Optiparts",
    },
    {
      "id": 2773,
      "name": "Windex (Sweden)",
    },
    {
      "id": 9901,
      "name": "Anton Scherbakov",
    },
    {
      "id": 11182,
      "name": "_jamuuk",
    },
    {
      "id": 11183,
      "name": "PJK",
    },
    {
      "id": 11185,
      "name": "Regate (LV)",
    },
    {
      "id": 11186,
      "name": "TommyyLyy",
    },
    {
      "id": 11187,
      "name": "Pro Marine Trade",
    },
    {
      "id": 11188,
      "name": "Татьяна Каверина (Armastus)",
    },
    {
      "id": 11189,
      "name": "NORA LII SOPRADE SELTS",
    },
    {
      "id": 11190,
      "name": "Andres",
    },
    {
      "id": 11191,
      "name": "Nautix",
    },
    {
      "id": 11192,
      "name": "Torivallavalitsus",
    },
    {
      "id": 11193,
      "name": "Ninne",
    },
    {
      "id": 11194,
      "name": "Urmas Varris",
    },
    {
      "id": 11195,
      "name": "WindWalker",
    },
    {
      "id": 11196,
      "name": "Via Grupp OU (Командор)",
    },
    {
      "id": 11201,
      "name": "Tõnis",
    },
    {
      "id": 11202,
      "name": "Jaanus",
    },
    {
      "id": 11203,
      "name": "Meister Arle OU",
    },
    {
      "id": 11204,
      "name": "Prime Marine Service OU",
    },
    {
      "id": 11205,
      "name": "Viking",
    },
    {
      "id": 11206,
      "name": "Saku",
    },
    {
      "id": 11207,
      "name": "SineneLind",
    },
    {
      "id": 11208,
      "name": "Alex Tido",
    },
    {
      "id": 11209,
      "name": "AZZURRA",
    },
    {
      "id": 11210,
      "name": "квас на пиво",
    },
    {
      "id": 11211,
      "name": "MIA (Tallin)",
    },
    {
      "id": 11212,
      "name": "MINNI",
    },
    {
      "id": 11213,
      "name": "Jonпgert OU",
    },
    {
      "id": 11214,
      "name": "MIA Pärnu",
    },
    {
      "id": 11217,
      "name": "Tauno Teluk",
    },
    {
      "id": 11219,
      "name": "Капитан",
    },
    {
      "id": 11220,
      "name": "Johannes  Atlandi Võlg",
    },
    {
      "id": 11221,
      "name": "Nimi",
    },
    {
      "id": 11222,
      "name": "Viahooldus ОÜ",
    },
    {
      "id": 11223,
      "name": "Kaupmees",
    },
    {
      "id": 11224,
      "name": "Reino",
    },
    {
      "id": 11225,
      "name": "Rubit OU",
    },
    {
      "id": 11226,
      "name": "Aleksey  Rusetsky",
    },
    {
      "id": 11227,
      "name": "Corinthian Events OÜ",
    },
    {
      "id": 11228,
      "name": "Ronstan DK",
    },
    {
      "id": 11229,
      "name": "Narva-Joesuu PurjeSpoordiKool",
    },
    {
      "id": 11230,
      "name": "Saaremaa Meri Spordiselts",
    },
    {
      "id": 11231,
      "name": "Tallux Orto",
    },
    {
      "id": 11232,
      "name": "Uku Kuusk",
    },
    {
      "id": 11234,
      "name": "Bainbridge Inc (UK)",
    },
    {
      "id": 11235,
      "name": "BetaMarine (UK)",
    },
    {
      "id": 11236,
      "name": "Change Consulting OÜ",
    },
    {
      "id": 11237,
      "name": "Rait Maru",
    },
    {
      "id": 11239,
      "name": "Žaklar OU",
    },
    {
      "id": 11240,
      "name": "Comar Systems Limited",
    },
    {
      "id": 11241,
      "name": "Kaidor",
    },
    {
      "id": 11242,
      "name": "MADEN",
    },
    {
      "id": 11243,
      "name": "Ropebox Eesti OÜ",
    },
    {
      "id": 11244,
      "name": "Muri",
    },
    {
      "id": 11245,
      "name": "Carrestoration OU",
    },
    {
      "id": 11246,
      "name": "Siren Sailing OU",
    },
    {
      "id": 11247,
      "name": "Episood OU",
    },
    {
      "id": 11248,
      "name": "Akster Grupp OU",
    },
    {
      "id": 11249,
      "name": "Alter",
    },
    {
      "id": 11250,
      "name": "Ronstan AUS",
    },
    {
      "id": 11251,
      "name": "Innon Finance OU",
    },
    {
      "id": 11252,
      "name": "Lettore OÜ",
    },
    {
      "id": 11253,
      "name": "Navideep Services OÜ",
    },
    {
      "id": 11254,
      "name": "Athaman (Polska)",
    },
    {
      "id": 11255,
      "name": "Lõbusõit OÜ",
    },
    {
      "id": 11256,
      "name": "Aleksei Mikhalev",
    },
    {
      "id": 11257,
      "name": "Ropebox OÜ",
    },
    {
      "id": 11258,
      "name": "Kaubakeskus OÜ",
    },
    {
      "id": 11259,
      "name": "Rein Ottoson OÜ",
    },
    {
      "id": 11260,
      "name": "Robert",
    },
    {
      "id": 11261,
      "name": "Merema OÜ",
    },
    {
      "id": 11262,
      "name": "Alexander Rusetsky",
    },
    {
      "id": 11263,
      "name": "Natalja Rusetskaya",
    },
    {
      "id": 11264,
      "name": "Jahtklubi Dago",
    },
    {
      "id": 11265,
      "name": "Elev Luik",
    },
    {
      "id": 11266,
      "name": "_ЗХ2023",
    },
    {
      "id": 11267,
      "name": "Vsevolod pyramidaprint",
    },
    {
      "id": 11268,
      "name": "Emerald",
    },
    {
      "id": 11270,
      "name": "Strand Sailing Management",
    },
    {
      "id": 11271,
      "name": "Tuule",
    },
    {
      "id": 11272,
      "name": "Meelis Kojamess (Баба Яга)",
    },
    {
      "id": 11273,
      "name": "Andrea FE64",
    },
    {
      "id": 11274,
      "name": "Reilut OÜ",
    },
    {
      "id": 11275,
      "name": "Pärnu cruises",
    },
    {
      "id": 11280,
      "name": "Madis Parts",
    },
    {
      "id": 11281,
      "name": "Tera ehitus OU",
    },
    {
      "id": 11282,
      "name": "Korkvender OU",
    },
    {
      "id": 11291,
      "name": "STRAND PROJEKT OÜ",
    },
    {
      "id": 11292,
      "name": "RTR Tehnika",
    },
    {
      "id": 11293,
      "name": "Eprodukt  OÜ",
    },
    {
      "id": 11294,
      "name": "Madis Parts",
    },
    {
      "id": 11295,
      "name": "SeaKayaking",
    },
    {
      "id": 11296,
      "name": "8 Motor Oy",
    },
    {
      "id": 11297,
      "name": "SKELTMER OÜ",
    },
    {
      "id": 11298,
      "name": "Jäälind",
    },
    {
      "id": 11299,
      "name": "MattChem (France)",
    },
    {
      "id": 11300,
      "name": "Car Restoration OU",
    },
    {
      "id": 11301,
      "name": "Valgeranna Seikluspark OÜ",
    },
    {
      "id": 11302,
      "name": "Kaif OÜ",
    },
    {
      "id": 11303,
      "name": "Julechka",
    },
    {
      "id": 11304,
      "name": "Longboat",
    },
    {
      "id": 11305,
      "name": "SAILING MASSIVE  OÜ",
    },
    {
      "id": 11306,
      "name": "AB Technology",
    },
    {
      "id": 11307,
      "name": "Norman Saarso",
    },
    {
      "id": 11308,
      "name": "Tuisu OÜ",
    },
    {
      "id": 11309,
      "name": "Fairaway",
    },
    {
      "id": 11311,
      "name": "Alibi OÜ",
    },
    {
      "id": 11312,
      "name": "Dirham motorpaad",
    },
    {
      "id": 11313,
      "name": "Niined group",
    },
    {
      "id": 11314,
      "name": "Magna Fide",
    },
    {
      "id": 11315,
      "name": "Корячкин",
    },
    {
      "id": 11317,
      "name": "Konstrado",
    },
    {
      "id": 11318,
      "name": "Vektom OÜ",
    },
    {
      "id": 11319,
      "name": "Kalev Jahtklubi",
    },
    {
      "id": 11320,
      "name": "Marta Vintage OÜ",
    },
  ],
  "orders": [
    {
      "allSum": 11.5,
      "clientName": "Щербаков Антон Васильевич",
      "date": "2022-07-10T12:34:00.000Z",
      "id": 204890,
      "isPaid": false,
      "paidSum": null,
      "payType": null,
      "positionCount": 3,
      "productsCount": -3,
      "productsSum": 7.5,
      "status": "продажа",
    },
  ],
  "pagesCount": 1,
  "rowsCount": 1,
  "totalAllSum": 11.5,
  "totalPaidSum": null,
  "totalProductSum": 7.5,
}
`;
